usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -261,3 +262,96 @@ def test_cluster_import_invalid(valid_session_credentials, cluster_id, status):
             Job status should be in status given by `status` parameter.
         """
     api.wait_for_job_status(job_id, status=status)
+
+
+"""@pylatest api/gluster.cluster_unmanage
+API-gluster: cluster_unmanage
+***************************
+
+.. test_metadata:: author fbalak@redhat.com
+
+Description
+===========
+
+Positive unmanage gluster cluster.
+"""
+
+
+@pytest.mark.cluster_unmanage_gluster
+def test_cluster_unmanage_valid(
+        valid_session_credentials, cluster_reuse, valid_trusted_pool_reuse):
+    """@pylatest api/gluster.cluster_unmanage
+        .. test_step:: 1
+
+            Check that tested cluster is correctly managed by Tendrl.
+
+        .. test_result:: 1
+
+            There is in Tendrl ``"is_managed":"yes"`` for cluster with id [cluster_id].
+            Graphite contains data related to health of tested cluster.
+
+        """
+    tendrl_api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    graphite_api = graphiteapi.ApiCommon()
+
+    cluster_id = cluster_reuse["cluster_id"]
+    pytest.check(
+        cluster_id is not None,
+        "Cluster id is: {}".format(cluster_id))
+    pytest.check(
+        cluster_reuse["is_managed"] == "yes",
+        "is_managed: {}\nThere should be ``yes``.".format(cluster_reuse["is_managed"]))
```

Comment:
Is this really needed? We could use the same test for a cluster which fails to be imported, hence it's not managed. However for first approach that's fine.
Anyway please think about that as both test cases are pretty similar. Only difference will be in those first checks 'is managed' check and 'cluster health' check. Some parameter and/or if statement makes this test two test case scenarios test.

Reply:
Ok, we can discuss this.